src/main/java/core/basesyntax/FileWriterImpl.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+
+public class FileWriterImpl implements FileWriter {
+    @Override
+    public void write(String data, String filePath) throws IOException {
```

Comment:
Avoid hardcoding file paths in your service implementations. Instead, accept the file path as a parameter to the method, which you are already doing. Ensure that the file path is provided by the caller, such as the `Main` class, and that it's a relative path to maintain portability.